src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj

```diff
@@ -18,6 +18,10 @@
     <Compile Include="$(CommonPath)System\ThrowHelper.cs" Link="Common\System\ThrowHelper.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == 'netstandard2.0'">
+    <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\NullableAttributes.cs" Link="System.Private.CoreLib\System\Diagnostics\CodeAnalysis\NullableAttributes.cs" />
+  </ItemGroup>
```

Comment:
It turns out this is redundant, as `NullableAttributes.cs` is already included in most libraries automatically:
https://github.com/dotnet/runtime/blob/1c824d929b734db420babbc04f44a1491d8a7efb/src/libraries/Directory.Build.targets#L170-L176

This explains https://github.com/dotnet/runtime/pull/101336#issuecomment-2067733988: 
> error CS0436: The type 'NotNullIfNotNullAttribute' in 'src\libraries\System.Private.CoreLib\src\System\Diagnostics\CodeAnalysis\NullableAttributes.cs' conflicts with the imported type 'NotNullIfNotNullAttribute' in 'Microsoft.Extensions.Configuration.Binder, ...'.

`Microsoft.Extensions.Configuration.Binder.SourceGeneration.Tests.csproj` has its own `NullableAttributes.cs` that is has ignored, but when actually used it conflicts with `internal` one in `Microsoft.Extensions.Configuration.Binder`:
https://github.com/dotnet/runtime/blob/1c824d929b734db420babbc04f44a1491d8a7efb/src/libraries/Microsoft.Extensions.Configuration.Binder/src/Properties/InternalsVisibleTo.cs#L7

There may be a cleaner way, but I was able to work around this by allowing projects to opt out of the import.